openadapt/app/objects/local_file_picker.py

```diff
@@ -0,0 +1,108 @@
+# retrieved from https://github.com/zauberzeug/nicegui/tree/main/examples/local_file_picker
+
+from pathlib import Path
+from typing import Dict, Optional
+
+from nicegui import ui
+
+
+class local_file_picker(ui.dialog):
```

Comment:
Please use `UpperCamelCase` convention for class names 🙏

Reply:
okay